src/dsl/verifier/verifier.ts

```diff
@@ -119,14 +120,14 @@ export class Verifier {
   // Run the Verification CLI process
   private runProviderVerification() {
     return (server: http.Server) => {
-      const port = (server.address() as AddressInfo).port;
+      const { port } = server.address() as AddressInfo;
       const opts: PactCoreVerifierOptions = {
         providerStatesSetupUrl: `${this.address}:${port}${this.stateSetupPath}`,
         ...omit(this.config, 'handlers'),
         providerBaseUrl: `${this.address}:${port}`,
         transports: this.config.transports?.concat([
           {
-            port: port,
+            port,
```

Comment:
I remember a time that the eslint plugin would just do these types of changes on save. I'm assuming that's what's happening in, in which case - huzzah!